src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/elementsdisputed/ElementsDisputedMidEventValidationHandler.java

```diff
@@ -68,15 +70,29 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
     
     private void checkAt38DocIsPresent(SscsCaseData sscsCaseData) {
         if ((!"uc".equalsIgnoreCase(sscsCaseData.getAppeal().getBenefitType().getCode())
-                && sscsCaseData.getDwpAT38Document() == null)
+                && documentsAt38IsMissing(sscsCaseData))
```

Comment:
I think also needs to check the existing sscsCaseData.getDwpAT38Document(), the new collection is for new cases